src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
+
+    public abstract void stopWork();
+}
+

```

Comment:
Redundant  empty line

Reply:
mvn causes an error if i remove this empty line